catroid/src/createatschool/java/org/catrobat/catroid/createatschool/ui/adapter/TemplateAdapter.java

```diff
@@ -110,14 +120,31 @@ public void onClick(View view) {
 	}
 
 	private void setImage(TemplateData templateData, ViewHolder holder) {
-		Picasso.with(context).load(templateData.templateResourceId).into(holder.image);
+		Picasso.with(context).load(getBaseUrl() + templateData.getThumbnail()).into(holder.image);
```

Comment:
this loads directly via HTTP? could there be problems if there is no internet connection?